packages/website/docs/plugins.md

```diff
@@ -3,21 +3,266 @@ id: plugins
 
-We'll also need to document the Plugin API.
+You can use one of our existing official plugins or build your own.
+
+## Usage
+
+### Using an Autocomplete plugin
+
+When using an Autocomplete plugin, all you need to do is provide it via the `plugins` option.
+
+For example, when using the Insights plugin, you can instantiate the plugin, then pass it down to your Autocomplete instance.
+
+```js {11,15}
+import algoliasearch from 'algoliasearch/lite';
+import { autocomplete } from '@algolia/autocomplete-js';
+import { createAlgoliaInsightsPlugin } from '@algolia/autocomplete-plugin-algolia-insights';
+import insightsClient from 'search-insights';
+
+const appId = 'latency';
+const apiKey = '6be0576ff61c053d5f9a3225e2a90f76';
+const searchClient = algoliasearch(appId, apiKey);
+insightsClient('init', { appId, apiKey });
+
+const algoliaInsightsPlugin = createAlgoliaInsightsPlugin({ insightsClient });
+
+autocomplete({
+  // ...
+  plugins: [algoliaInsightsPlugin],
+});
+```
+:::note
+
+Plugins execute sequentially, in the defined order.
+
+:::
+
+### Building your own plugin
+
+An Autocomplete plugin is a simple object that implements the `AutocompletePlugin` interface.
+
+It can provide sources, react to state changes, and hook into various autocomplete lifecycle steps. It has access to setters, including the [Context API](context), allowing it to store and retrieve arbitrary data at any time.
+
+Let's create a plugin that adds a static list of GitHub repositories to the autocomplete results.
+
+```js
+const topGitHubRepositoriesPlugin = {
+  getSources() {
+    return [
+      {
+        getItems() {
+          return [
+            { name: 'algoliasearch-client-javascript', stargazersCount: 884 },
+            { name: 'algoliasearch-client-php', stargazersCount: 554 },
+            { name: 'hn-search', stargazersCount: 383 },
+          ];
+        },
+        getItemUrl({ item }) {
+          return `https://github.com/algolia/${item.name}`;
+        },
+        templates: {
+          header() {
+            return 'Discover our top 3 GitHub repositories';
+          },
+          item({ item }) {
+            return item.name;
+          },
+        },
+      },
+    ];
+  },
+};
+
+autocomplete({
+  // ...
+  plugins: [topGitHubRepositoriesPlugin],
+});
+```
+
+Now, if you're willing to package and distribute your plugin for other people to use, you might want to expose a function instead.
+
+```js
+function createTopGitHubRepositoriesPlugin({ repositories, username }) {
+  const sortedRepositories = repositories.sort(
+    (a, b) => b.stargazersCount - a.stargazersCount
+  );
+
+  return {
+    getSources() {
+      return sortedRepositories;
```

Comment:
This should be in `getItems` (see example above).

Reply:
True 🤦‍♀️ Fixed in 101ad0c.